modules/call-dispatch/src/lib.rs

```diff
@@ -172,6 +177,8 @@ decl_event!(
 		MessageDispatched(InstanceId, MessageId, DispatchResult),
 		/// We have failed to decode Call from the message.
 		MessageCallDecodeFailed(InstanceId, MessageId),
+		/// The call from the message has been rejected by the call filter.
+		MessageCallRejected(InstanceId, MessageId),
```

Comment:
Perhaps something like `MessageCallFiltered` in order to make the reason for rejecting the message more explicit?

Reply:
I have nothing against that, except for the fact we have similar `MessageRejected` event, which rejects "filtered" message before it reaches dispatch mechanism. If you think `MessageCallRejected` it's better - just respond here, I'll change the name.

Reply:
I wasn't aware of `MessageRejected`. If we basically use 'rejected' for 'filtered by some means', then `MessageCallRejected` is better, since it is more consistent.